AccountingSystem/ConfigTemplate.cfg

```diff
@@ -25,4 +25,14 @@ Services
       }
     }
   }
-}
\ No newline at end of file
+}
+Agents
+{
+  NetworkAgent
+  {
+    MaxCycles = 0
```

Comment:
Restarting, shutting down, etc. by operators is perfectly fine. With MaxCycles = 0 I just want to guarantee that when the agent is started it will work indefinitely long .